window-provider/wallet-connection-handlers.ts

```diff
@@ -469,4 +497,8 @@ export default function monitorForWalletConnectionPrompts(): void {
       observeMutations(hostnameToHandler[hostname])
     }
   })
+
+  // Commenting this out for now, because we don't have a way to put this behind a feature flag
+  // SUPPORT_WALLET_CONNECT flag
+  // observeMutations(addTallyButtonForWalletConnectModal)
```

Comment:
Let's read the feature flag directly for now

```suggestion
  if (process.env.SUPPORT_WALLET_CONNECT === "true")
    observeMutations(addTallyButtonForWalletConnectModal)
```